charts/kueue/values.yaml

```diff
@@ -0,0 +1,85 @@
+# Default values for kueue.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+nameOverride: ""
+fullnameOverride: ""
+
+# Enable each function, like kustomize https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml
+enablePrometheus: false
+enableCertManager: true
+
+# Customize controlerManager
+controllerManager:
+  kubeRbacProxy:
+    image: 
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
+      # tag
+      tag: v0.8.0
+      # This should be set to 'IfNotPresent' for released version
+      pullPolicy: IfNotPresent
+  manager:
+    image:
+      repository: gcr.io/k8s-staging-kueue/kueue
+      # tag, if defined will use the given image tag, else Chart.AppVersion will be used
+      tag: main
```

Comment:
we will also need to update this file when doing a release, right?

Reply:
Yes, you are right.
If we want to use a released image, we need to change the tag to a specific released version.
I've added this point to NEW_RELEASE.md as well.